README

```diff
@@ -204,9 +204,12 @@ users should not rely on the default origin and appid ("pam://$HOSTNAME")
 but set those parameters explicitly to the same value.
 
 == Individual authorization mapping by user
-Each user creates a `~/.config/Yubico/u2f_keys` file inside of their home
-directory and places the mapping in that file, the file must have only one
-line:
+Each user creates a `.config/Yubico/u2f_keys` (default) file inside their 
```

Comment:
This PR is about the debug_file not being created so I think it would be good to not include other significant changes if you want this to be merged (just a tip for any other PRs you might want to submit)

Reply:
Good point. I wanted to PR this config file issue first but somehow I failed to make github do my biddings. It just combined both patches in one single PR. How do I split this into two distinct PRs?

Reply:
Can you cherry-pick the commits regarding the individual user files in the PR? Or do you need me to cancel the PR and re-PR with only the right commits?

Reply:
What do prefer @a-dma ?

Reply:
I'm fine with cherry-picking the commits, but it's probably nicer if you just amend the PR so it shows as a self-standing contribution from you (rather than just a single random commit).

You can even modify the branch and force-push it if you don't want to close this PR and open a new one.

Reply:
@a-dma Thank you. 

> You can even modify the branch and force-push it if you don't want to close this PR and open a new one.

Done that.